examples/article.amp.html

```diff
@@ -162,7 +162,7 @@
             srcset="img/hero@1x.jpg 1x, img/hero@2x.jpg 2x"
             layout="responsive" width="360" placeholder
             alt="Curabitur convallis, urna quis pulvinar feugiat, purus diam posuere turpis, sit amet tincidunt purus justo et mi."
-            height="216" on="tap:headline-img-lightbox">
+            height="216" on="tap:headline-img-lightbox" role="img" tabindex="0">
```

Comment:
Hmm. Generally, `amp-img` should definitely add `role=img` automatically. If not - we should fix. However, what I was trying to say is that `role=img` doesn't really help with `tap` action here, since it doesn't make it clear that this element is actually tappable. I wonder what's the most appropriate way to mark this up. Seems that `role=button` is most appropriate in this case.